app/models/probate_case_fee.rb

```diff
@@ -0,0 +1,5 @@
+class ProbateCaseFee < Base
+  attribute :answer, Boolean
+
+  validates :answer, inclusion: { in: [true, false] }
+end
```

Comment:
Doesn't this need name and dob of deceased too?

[Link](http://ministryofjustice.github.io/fr-prototypes/public/journey-7/probate.html)